docs/config/_default/config.toml

```diff
@@ -0,0 +1,69 @@
+# when deploying to prod, use baseURL = "https://www.guardicore.com/infectionmonkey/docs"
+# baseURL = "https://staging-covuyicu.kinsta.cloud/infectionmonkey/docs/"
+baseURL = ""
+languageCode = "en-us"
+title = "Guardicore Infection Monkey - Documentation Hub"
+
+theme = "learn"
+
+[params]
+  # Change default color scheme with a variant one. See it in the themes folder.
+  themeVariant = "monkey"
+  # Prefix URL to edit current page. Will display an "Edit this page" button on top right hand corner of every page.
+  editURL = "https://github.com/guardicore/monkey/edit/develop/docs/content/"
```

Comment:
on production this link is broken in multiple pages

Reply:
Once we merge this PR this link ought to work